ci/infra/azure/variables.tf

```diff
@@ -0,0 +1,175 @@
+variable "azure_location" {
+  type = string
+  # default     = "West Europe"
+  description = "Name of the AZURE location to be used"
```

Comment:
I would be fine to disable the default, if that's the case drop the commented line and update the description to be something like: `Name of the AZURE location to be used (eg: "West Europe")`

Reply:
changed